apps/src/StudioApp.js

```diff
@@ -437,34 +429,6 @@ StudioApp.prototype.init = function (config) {
     config.showInstructionsWrapper(() => {});
   }
 
-  // In embed mode, the display scales down when the width of the
-  // visualizationColumn goes below the min width
-  if (config.embed && config.centerEmbedded) {
-    var resized = false;
-    var resize = function () {
-      var vizCol = document.getElementById('visualizationColumn');
-      var width = vizCol.offsetWidth;
-      var height = vizCol.offsetHeight;
-      var displayWidth = DEFAULT_MOBILE_NO_PADDING_SHARE_WIDTH;
-      var scale = Math.min(width / displayWidth, height / displayWidth);
-      var viz = document.getElementById('visualization');
-      viz.style['transform-origin'] = 'left top';
-      viz.style['-webkit-transform'] = 'scale(' + scale + ')';
-      viz.style['max-height'] = (displayWidth * scale) + 'px';
-      viz.style.display = 'block';
-      vizCol.style.width = '';
-      vizCol.style.maxWidth = displayWidth + 'px';
-      // Needs to run twice on initialization
-      if (!resized) {
-        resized = true;
-        resize();
-      }
-    };
-    // Depends on ResizeSensor.js
-    var ResizeSensor = require("./third-party/ResizeSensor");
-    ResizeSensor(document.getElementById('visualizationColumn'), resize);
-  }
-
```

Comment:
Note that this entire block of code was unused because `config.centerEmbedded` was never true.